solutions/security/ai/attack-discovery.md

```diff
@@ -3,9 +3,9 @@ mapped_pages:
   - https://www.elastic.co/guide/en/security/current/attack-discovery.html
   - https://www.elastic.co/guide/en/serverless/current/attack-discovery.html
 applies_to:
-  stack: preview
+  stack: ga
   serverless:
-    security: preview
+    security: ga
```

Comment:
Was this already GA in 9.0? If it's becoming GA in 9.1, it should be:

stack: preview 9.0, ga 9.1
serverless:
  security: ga